download/releaseNotes/releaseNotes7.adoc

```diff
@@ -5,6 +5,27 @@
 :awestruct-release_notes_version: 7
 :awestruct-release_notes_version_qualifier: Final
 
+== New and noteworthy: Engine 7.27.0.Final (NOT YET RELEASED)
+
+=== Pillar moves are faster
+
+`PillarSwapMove` and `PillarChangeMove` have seen improvements under the hood and are now faster. We have observed
```

Comment:
Use active voice ( https://developers.google.com/style/voice )
X and Y have seen improvements under the hood => X and Y improved under the hood
We have observed => We observed

We usually don't do a space before percentage, but not sure what the styleguides are there...

Reply:
Wrt. the percentage: https://english.stackexchange.com/questions/3281/should-there-be-a-space-before-a-percent-sign

There is no consensus. But it is a good point that you usually write a space between a number and its unit. Therefore, a space between makes sense.

Reply:
I usually don't actually, IIRC :) so most of the other release notes use 5% etc IIRC.
Either way, I don't want to bikeshed this as there is no English requirement.